docs/en/pid-eaa-data-model.rst

```diff
@@ -149,20 +156,24 @@ The ``record`` MUST have at least the following sub parameters:
     - **Description**
     - **Reference**
   * - **type** 
-    - identification of the national eID framework used by the User. For example ``eidas.it.cie`` means that the CIE id identification scheme is used by the User. 
+    - Identification of the trust framework used for obtaining the verified claims. For example, in case of PID, ``eidas.it.cie`` means that the CIE id identification scheme is used by the User. 
```

Comment:
```suggestion
    - It uniquely identifies the identification scheme used for the provisioning of the credential. For example, in case of PID, ``eidas.it.cie`` means that the CIE id identification scheme is used. 
```

Reply:
done. I kept "trust framework",  as it is more general and it better applies to a (Q)EAA.